l10n_ch_hr_payroll/models/hr_contract.py

```diff
@@ -9,8 +10,8 @@
 class HrContract(models.Model):
     _inherit = 'hr.contract'
 
-    holiday_rate = fields.Float(string='Holiday Rate')
-    lpp_rate = fields.Float(string='LPP Rate',
+    holiday_rate = fields.Float(string='Holiday Rate', translate=True)
+    lpp_rate = fields.Float(string='OBP Rate', translate=True,
                             digits=dp.get_precision('Payroll Rate'))
-    lpp_amount = fields.Float(string='LPP Amount',
+    lpp_amount = fields.Float(string='OBP Amount', translate=True,

```

Comment:
No reason to put a fload field as translatable the label will be alwals translatable